var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -557,23 +557,12 @@ def headers(self):
     def setup_build_tests(self):
         """Copy the build test files after the package is installed to an
         install test subdirectory for use during `spack test run`."""
-        self.cache_extra_test_sources('src/ksp/ksp/tutorials')
-        self.cache_extra_test_sources('src/snes/tutorials')
+        self.cache_extra_test_sources(join_path('src', 'ksp', 'ksp', 'tutorials'))
+        self.cache_extra_test_sources(join_path('src', 'snes', 'tutorials'))
```

Comment:
Suggest making a single call to the method for both paths:

```suggestion
        self.cache_extra_test_sources([
            join_path('src', 'ksp', 'ksp', 'tutorials'),
            join_path('src', 'snes', 'tutorials')
        ])
```